README.md

```diff
@@ -1,71 +1,83 @@
-- Ruby
-- irb
-- CMD with Ruby
-- rspec
-- sql
--postgres
-
-## Getting Started
-
-To get a copy of this project running on your local computer, do the followings:
-
-### Prerequisites
-- Must have a working computer
-- A Web-Browser installed (Chrome - Recommended)
-- A code editor (VSCode - Recommended)
-
-### Install
-
-Follow all steps to install [_Ruby_](https://rubyinstaller.org/) 
-
-### Setup
-
-- Open the terminal on your computer
-- Navigate through directories using [change directory (cd)](https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10) command to where you want to clone the repository.
-
-- **Type the following lists of commands and press "ENTER" after each command:**
-
-- **Open folder in VSC**
-
-  - code .
-
-  - _Install dependencies locally_
-    - bundle install
-
-
-## Authors
-
-👤 **Abdelrhman Samy Saad**
-
-- GitHub: [@Mov305](https://github.com/Mov305)
-- Twitter: [@Mov305](https://twitter.com/Mov_abd)
-- LinkedIn: [@Mov305](https://www.linkedin.com/in/abdelrhman-samy-80b14b215/)
-
-👤 **Oluyemi Paul**
-
-- GitHub: [@Ol-create](https://www.github.com/Ol-create)
-- Twitter: [@OluyemiPaul99](https://twitter.com/OluyemiPaul99)
-- LinkedIn: [@paul-oluyemi](https://linkedin.com/in/paul-oluyemi)
-
-
-
-## Acknowledgements ⭐️
-
-- Special thanks to the team for being fully commited on finishing this project together.
-
-## Show your support
-
-- Give a ⭐️ if you like this project!
-
-## 📝 License
-
-- This project is [MIT](./LICENSE) licensed.
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![Ruby](https://img.shields.io/badge/-Ruby-red)
+
+# Ruby Capstone Project
+
+> This is a Ruby Capstone Project. The project is about building a command-line application that helps to manage a library. The application allows users to add books, authors, and genres to the library. Users can also list all books, authors, and genres in the library. The application also allows users to list all books by a specific author or genre.
+
+[Link for presentation](https://drive.google.com/file/d/12eRAhOSMyqIQRY9jOxehz_Q9dsMz_9N4/view?usp=share_link
+)
+
+
+## Built With
+- Ruby
+- irb
+- CMD with Ruby
+- rspec
+- sql
+-postgres
+
+## Getting Started
+
+To get a copy of this project running on your local computer, do the followings:
+
+### Prerequisites
+- Must have a working computer
+- A Web-Browser installed (Chrome - Recommended)
+- A code editor (VSCode - Recommended)
+
+### Install
+
+Follow all steps to install [_Ruby_](https://rubyinstaller.org/) 
+
+### Setup
+
+- Open the terminal on your computer
+- Navigate through directories using [change directory (cd)](https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10) command to where you want to clone the repository.
+
+- **Type the following lists of commands and press "ENTER" after each command:**
+
+- **Open folder in VSC**
+
+  - code .
+
+  - _Install dependencies locally_
+    - bundle install
+

```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a hint/command line in your README that shows how to run your application. This will help users to know how your application works.